docs/source/standard.md

```diff
@@ -15,137 +15,364 @@ The SATRE standard is based around the idea of capabilities, some of which are r
 Any particular TRE implementation should be able to score itself against each capability as either "supported", "partially supported" or "unsupported" (see {ref}`evaluation` for details).
 The capabilities are grouped into the following broad categories:
 
-- information governance
-- computing technology
-- data management
-- information security
+- {ref}`information governance <standard_capability_information_governance>`
```

Comment:
FYI markdown style links `[information governance](standard_capability_information_governance)` should also work for MyST cross-refs

Reply:
Do you prefer markdown links? I'm happy to use those instead.

Reply:
I prefer it since it means there's less non-markdown syntax, but that's not a blocker here. We can deal with it later.